tests/tests/test_filetransfer.py

```diff
@@ -630,9 +647,7 @@ def test_filetransfer_limits_download_err_not_a_regular_file(
         r = download_file(path, self.devid, self.auth_token)
 
         assert r.status_code == 400, r.json()
-        assert (
-            r.json().get("error") == "file transfer failed: path is not a regular file"
-        )
+        assert r.json().get("error").endswith("path is not a regular file")
```

Comment:
This is the only change. The rest is just formatting :see_no_evil:

Reply:
Unfortunately CI did not like your reformatting :robot: